next.config.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-env node */
+
+const withSourceMaps = require('@zeit/next-source-maps')
```

Comment:
I see `@zeit/next-source-maps` was also added here. Is this specifically just a nice addition for Sentry logs/traces?

Reply:
Source maps is a mechanism to map bundled and minified code with the original source files. If we just run sentry as it is, it will show errors like: `You nave an error on line 1, symbol 3454560 in the gX method.` Source maps will ideally show us original file, line and method name. (But right now it's looks like it didn't work correctly)

It's not strictly related to Sentry and is used for example by browser Dev Tools, but it make using Sentry more convenient.

Reply:
Right, I did read about source maps and it seems like a great feature. I was just wondering if this was added specifically for Sentry traces or whether it can also be used on the Browser now.

Reply:
Well, we should be, because I leave them enabled (there is also an option to send them directly to sentry through API and not publish on site). But in practice from that I see, I misconfigured them somehow and didn't see them anywhere.

Reply:
OK cool please let us know when you figure it out 🙂